src/engraving/draw/drawtypes.h

```diff
@@ -38,6 +38,33 @@ enum class PolygonMode {
     Convex,
     Polyline
 };
+
+enum PenStyle { // pen style
```

Comment:
better add `enum class ...` and use `PenStyle::NoPen` and etc

Reply:
Also for other enums

Reply:
@igorkorsukov is it fine to leave like this until we don't have Qt? Otherwise I have conflicts with Qt namespace and I don't want to rename or add extra namespace

Reply:
Did not understand you. enum class, on the contrary, excludes conflicts